### ISSUE TITLE
Unify line height.

### ISSUE THREAD

**Issue by notlee** [Financial-Times/o-layout#22]
We use a custom line-height in `o-layout` that differs to `o-table`.

Maybe..?
- Line height shouldn't be unique (probably not, for readability of techdocs)
- `o-layout` should override `o-table`
- `o-table` should inherit font attributes

Link in body:
![image](https://user-images.githubusercontent.com/10405691/44096001-17c8b608-9fd2-11e8-841c-771e74bdfb4c.png)

Link nested in table:
![image 1](https://user-images.githubusercontent.com/10405691/44096007-1c5fd71e-9fd2-11e8-874f-50c1c4de3f1f.png)

**Comment by gvonkoss** [comment]
I agree, and I'm happy to add this to the breaking change that we'll be making soon? I vote o-layout should override o-table for now, if we make o-table inherit, it may cause unwanted/unexpected changes elsewhere.

**Comment by gvonkoss** [comment]
We've changed our minds because there will be a breaking of `o-table` soon which can include the font feature inheritance.

This also means that we leave specific styling out of `o-layout`, and don't have to chase down small edge cases and minor tweaks further down the line.

**Comment by gvonkoss** [comment]
we did?

**Comment by gvonkoss** [comment]
indeed we did

**Comment by notlee** [comment]
Thought: Now we can our typographic scale per brand, perhaps the line height for the internal brand should be updated across the board rather than just within `o-layout`.
